lib/pure/asyncftpclient.nim

```diff
@@ -253,15 +253,18 @@ proc retrText*(ftp: AsyncFtpClient, file: string): Future[string] {.async.} =
 
   result = await ftp.getLines()
 
-proc getFile(ftp: AsyncFtpClient, file: File, total: BiggestInt,
+proc getFile(ftp: AsyncFtpClient, dest: string, total: BiggestInt,
```

Comment:
Please keep changes to a minimum. This is one in a series of changes which I don't think are relevant to this PR.

Could you please clean it up so that the PR is as small as possible?